package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.95.0",
+  "version": "2.97.0",
```

Comment:
accidentally did a double bump :hide:

Reply:
Not a big deal. Though you could undo the commit if you wanted to avoid skipping a version. It looks like someone else just published 2.96.0, so it worked out anyway 😄